lib/xcodeproj/project/object/helpers/file_references_factory.rb

```diff
@@ -178,7 +178,8 @@ def new_subproject(group, path, source_tree)
             ref = new_file_reference(group, path, source_tree)
             ref.include_in_index = nil
 
-            product_group_ref = find_products_group_ref(group, true)
+            product_group_ref = group.project.new(Xcodeproj::Project::PBXGroup)
```

Comment:
Probably could be just
```suggestion
            product_group_ref = group.project.new(PBXGroup)
```